bitnami/scylladb/templates/statefulset.yaml

```diff
@@ -0,0 +1,676 @@
+          {{- else }}
+          command:
+            - /opt/bitnami/scylladb/supervisor/scylla-jmx.sh
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.jmxProxy.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.jmxProxy.args "context" $) | nindent 12 }}
+          {{- end }}
+          image: {{ include "scylladb.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.jmxProxy.containerSecurityContext.enabled }}
+          securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.jmxProxy.containerSecurityContext "context" $) | nindent 12 }}
+          {{- end }}
+          env:
+            {{- if or .Values.jvm.maxHeapSize (include "scylladb.memory.getLimitInM" .) }}
+            - name: MAX_HEAP_SIZE
+              value: {{ include "scylladb.memory.calculateMaxHeapSize" . | quote }}
+            {{- end }}
+            {{- if or .Values.jvm.newHeapSize (include "scylladb.memory.getLimitInM" .) }}
+            - name: HEAP_NEWSIZE
+              value: {{ include "scylladb.memory.calculateNewHeapSize" . | quote }}
+            {{- end }}
+            {{- if .Values.jvm.extraOpts }}
+            - name: JVM_EXTRA_OPTS
+              value: {{ .Values.jvm.extraOpts | quote }}
+            {{- end }}
+            - name: SCYLLA_JMX_PORT
+              value: {{ printf "-jp %d" (int .Values.containerPorts.jmx) | quote }}
+            - name: SCYLLA_API_PORT
+              value: {{ printf "-p %d" (int .Values.containerPorts.api) | quote }}
+            {{- if .Values.jmxProxy.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.jmxProxy.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            {{- if .Values.jmxProxy.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.jmxProxy.extraEnvVarsCM "context" $) }}
+            {{- end }}
+            {{- if .Values.jmxProxy.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.jmxProxy.extraEnvVarsSecret "context" $) }}
+            {{- end }}
+          {{- if not .Values.diagnosticMode.enabled }}
+          {{- if .Values.jmxProxy.customLivenessProbe }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.jmxProxy.customLivenessProbe "context" $) | nindent 12 }}
+          {{- else if .Values.jmxProxy.livenessProbe.enabled }}
+          livenessProbe:
+            exec:
+              command:
+                - pgrep
+                - -f
+                - scylla-jmx
```

Comment:
Using the same probes in `scylladb-jmx-proxy` too

Reply:
Using the nodetool command as this the proxy is necessary for nodetool to work (even though the port is on the other node)